I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script_business.rpy", line 267, in <module>
    scene black with fade
NameError: name 'mid_sleft' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "lib/windows-i686/script_business.rpyc", line 267, in script
  File "C:\renpy-6.99.11-sdk\renpy\ast.py", line 1064, in execute
    show_imspec(self.imspec, atl=getattr(self, "atl", None))
  File "C:\renpy-6.99.11-sdk\renpy\ast.py", line 1025, in show_imspec
    at_list = [ renpy.python.py_eval(i) for i in at_list ]
  File "C:\renpy-6.99.11-sdk\renpy\python.py", line 1670, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\renpy-6.99.11-sdk\renpy\python.py", line 1665, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script_business.rpy", line 267, in <module>
    scene black with fade
NameError: name 'mid_sleft' is not defined

Windows-8-6.2.9200
Ren'Py 6.99.11.1749
Shigoto to Koi 0.0.0
